android/library/src/main/java/com/liferay/mobile/screens/ddl/model/DDMStructure.java

```diff
@@ -73,6 +100,8 @@ protected DDMStructure(Parcel in, ClassLoader classLoader) {
         Parcelable[] array = in.readParcelableArray(Field.class.getClassLoader());
         fields = new ArrayList(Arrays.asList(array));
         locale = (Locale) in.readSerializable();
+        name = in.readString();
+        description = in.readString();
```

Comment:
new fields should be added here, right?

Reply:
Yep, I fixed the parcelable implementation in the last commits.